src/webview_provider/datapilotPanel.ts

```diff
@@ -107,74 +108,79 @@ export class DataPilotPanel extends AltimateWebviewProvider {
           project: this.queryManifestService.getProject(),
         });
         break;
+
       case "docgen:insert":
+      case "testgen:insert":
         this.emitterService.eventEmitter.fire({
-          command: "docgen:insert",
+          command: command,
           payload: params,
         });
         break;
-      case "queryAnalysis:explain":
-        try {
-          const response = await this.queryAnalysisService.executeQueryAnalysis(
-            params,
-            QueryAnalysisType.EXPLAIN,
-            syncRequestId,
-          );
 
-          this.sendResponseToWebview({
-            command: "response",
-            data: {
-              response,
-            },
-            syncRequestId,
-          });
-        } catch (err) {
-          this.sendResponseToWebview({
-            command: "response",
-            syncRequestId,
-            error: (err as Error).message,
-          });
-        }
+      case "queryAnalysis:explain":
```

Comment:
used `handleSyncRequestFromWebview` instead of using try catch implementation for each command